packages/server/src/models/Category.ts

```diff
@@ -95,12 +95,19 @@ CategorySchema.statics.getCategoryList = async function(): Promise<
 };
 
 CategorySchema.statics.getByStub = async function(
-  stub: string
+  stub: string,
+  includeDeletedResources: boolean = false
```

Comment:
I did add this parameter incase we want to add a query string that enables fetching trashed resources as well.